.gitignore

```diff
@@ -28,6 +28,7 @@ tools/ido5.3_compiler/
 tools/ido7.1_compiler/
 tools/qemu-mips
 tools/ido_recomp/* binary
+ctx.c
```

Comment:
Really glad this was added. The amount of times I've committed the context by accident...